fix(minifier): remove `continue` in the end of for-in / for-of

sapphi-red opened the pull request:
Compress
```js
for (a of b) { c(); continue; }
for (a in b) { c(); continue; }
```
to
```js
for (a of b) { c(); }
for (a in b) { c(); }
```

Boshen commented:
### Merge activity

* **Oct 1, 4:55 AM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

sapphi-red committed:
fix(minifier): remove `continue` in the end of for-in / for-of (#14186)

Compress
```js
for (a of b) { c(); continue; }
for (a in b) { c(); continue; }
```
to
```js
for (a of b) { c(); }
for (a in b) { c(); }
```
